playground/infrastructure/cloudbuild/examples_ci.sh

```diff
@@ -0,0 +1,244 @@
+# COMMIT - Git commit hash to build containers from (Default: HEAD)
+# DIFF_BASE - Git branch to compare with $COMMIT to look for changed examples (Default: origin/master)
+# SDKS - Array of SDKS to validate (Default: java python go)
+# ORIGIN - examples origin (Default: PG_EXAMPLES)
+# ALLOWLIST - List of paths (relative to the repo root) not in examples (SUBDIRS) that cause reb
+
+for ARGUMENT in "$@"
+do
+   KEY=$(echo $ARGUMENT | cut -f1 -d=)
+
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+
+   export "$KEY"="$VALUE"
+done
+
+export LOG_PATH=${LOG_PATH-"/dev/null"}
+export BEAM_ROOT_DIR=${BEAM_ROOT_DIR-"/workspace/beam"}
+export PROJECT_ID=${PROJECT_ID}
+export BEAM_VERSION=${BEAM_VERSION-"2.44.0"}
+export SUBDIRS=${SUBDIRS-"./learning/katas ./examples ./sdks"}
+export SDKS=${SDKS-"java python go"}
+export COMMIT=${COMMIT-"HEAD"}
+export DIFF_BASE=${DIFF_BASE-"origin/master"}
+export BEAM_CONCURRENCY=${BEAM_CONCURRENCY-"4"}
+export ORIGIN=${ORIGIN-"PG_EXAMPLES"}
+export ALLOWLIST=${ALLOWLIST-"playground/infrastructure playground/backend"}
+
+function LogOutput ()
+{
+    echo "$(date --utc '+%D %T') $1" >> $LOG_PATH
+    # CILOG keyword to simplify search over the global log
+    echo "CILOG $(date --utc '+%D %T') $1"
+}
+
+
+LogOutput "Input variables:
+            LOG_PATH=$LOG_PATH
+            BEAM_ROOT_DIR=$BEAM_ROOT_DIR
+            PROJECT_ID=$PROJECT_ID
+            BEAM_VERSION=$BEAM_VERSION
+            SUBDIRS=$SUBDIRS
+            SDKS=$SDKS
+            COMMIT=$COMMIT
+            BEAM_CONCURRENCY=$BEAM_CONCURRENCY
+            ORIGIN=$ORIGIN
+            ALLOWLIST=$ALLOWLIST"
+
+# Assigning constant values
+# Script starts in a clean environment in Cloud Build. Set minimal required environment variables
+if [ -z "$PATH" ]; then
+    export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin/:/usr/bin:/sbin:/bin"
+fi
+if [ -z "$HOME" ]; then
+    export HOME="/builder/home"
+fi
+
+export STEP=CI
+export SDK_CONFIG="$BEAM_ROOT_DIR/playground/sdks.yaml"
+export BEAM_EXAMPLE_CATEGORIES="$BEAM_ROOT_DIR/playground/categories.yaml"
+export GRADLE_VERSION=7.5.1
+export GO_VERSION=1.18
+
+LogOutput "Installing python java8 and dependencies"
+apt-get update > /dev/null
+apt update > /dev/null
+export DEBIAN_FRONTEND=noninteractive
+
+LogOutput "Installing Python environment"
+apt-get install -y apt-transport-https ca-certificates software-properties-common curl unzip apt-utils > /dev/null
+add-apt-repository -y ppa:deadsnakes/ppa > /dev/null && apt update > /dev/null
+apt install -y python3.8 python3.8-distutils python3-pip > /dev/null
+apt install -y --reinstall python3.8-distutils > /dev/null
+pip install --upgrade google-api-python-client > /dev/null
+python3.8 -m pip install pip --upgrade > /dev/null
+ln -s /usr/bin/python3.8 /usr/bin/python > /dev/null
+apt install -y python3.8-venv > /dev/null
+pip install -r playground/infrastructure/requirements.txt > /dev/null
```

Comment:
This whole section seems to be exactly the same as in the other script - can we move it some common place?

Reply:
Let's keep it as is for now. It's improvement for future.